README.md

```diff
@@ -29,9 +39,8 @@ Value is X-JWT-Assertion.
 
 Example
 
-```
-import byu_jwt
-current_jwt_header = byu_jwt.BYU_JWT_HEADER_CURRENT
+```python
+current_jwt_header = byujwt.BYU_JWT_HEADER_CURRENT
```

Comment:
I'm going to make notes here even though I know this is code oriented. I feel like we should change this to `BYU_JWT_HEADER` and then leave the other as `BYU_JWT_HEADER_ORIGINAL`. My reasoning being because most people won't understand the 'current' connotation as they don't care about the original, the use of the original JWT is far from the norm at the moment, so I think focusing on the most used happy path could be nice.

Reply:
Resolved